server/conf/i18n/messages

```diff
@@ -349,16 +349,16 @@ button.continueEditing=Continue editing
 button.exitApplication=Exit application
 
 
-#----------------------------------------------------------------------------------------#
-# Error - error info page when user tries to access a disabled program via its deep link #
-#----------------------------------------------------------------------------------------#
+#-------------------------------------#
+# Error - Disabled program error page #
```

Comment:
**suggestion:** Sorry I meant the first part. So `Disabled program error page - error info page when user tries to access a disabled program via its deep link`